Apps/Database/src/Migrations/GatewayDbContextModelSnapshot.cs

```diff
@@ -3428,6 +3548,23 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                         .IsRequired();
                 });
 
+            modelBuilder.Entity("HealthGateway.Database.Models.DependentAudit", b =>
+                {
+                    b.HasOne("HealthGateway.Database.Models.Dependent", "Dependent")
+                        .WithMany("DependentAudits")
+                        .HasForeignKey("DelegateHdId")
+                        .OnDelete(DeleteBehavior.SetNull)
+                        .IsRequired();
```

Comment:
Similar to above, is it right to set IsRequired() if nullable will be a valid value on deleting dependent?